README.md

```diff
@@ -33,6 +33,13 @@ generated output.
 Including files manually like this lets us specify the position they appear in
 the page.
 
+## Making functional changes
+
+The GDS Way is built from the [Tech Docs Template](https://github.com/alphagov/tech-docs-template)
+repository. Any functional changes and bug fixes should be made here first, then follow the
```

Comment:
In the phrase, "any functional changes and bug fixes should be made here first", it reads like "here" means the GDS Way - because that's the site the user will be reading. Perhaps it should say, "changes and bug fixes should be made to that project first".

Reply:
Agree @pgchamberlin, updated accordingly.